src/components/Book.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { CircularProgressbar } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
+
+class Book extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+    this.value = 0.64;
+  }
+
+  render() {
+    const { title, author } = this.props;
+    return (
+      <div className="container my-3 px-5 border border-opacity-10 border-dark rounded-1 bg-light d-flex justify-content-between align-items-center flex-fill">
+        <div style={{ width: '30%' }}>
```

Comment:
- [ ] Let's try to avoid inline styling as it is not best practice. Kindly move all styles to an external CSS file.